### ISSUE TITLE
Desktop Client on Linux Mint 20 Mate doesn‘t start after upgrading to 3.1.0

### ISSUE THREAD

**Issue by rainer23** [nextcloud/desktop#2750]
**Expected behaviour**

Trouble-free start of the Nextcloud client software v 3.1 after upgrading from 3.0.3

**Actual behaviour**

After upgrading to 3.1.0 the client software starts, shows the logo in the system tray for about 3 seconds and then stops, no further action. In the terminal i’ve got the message qt5ct: using qt5ct plugin Speicherzugriffsfehler (Speicherabzug geschrieben). 

**Steps to reproduce**

1. Removing all Nextcloud packages via Synaptic
2. Re-installing via Synaptic from PPA
3. Same behaviour

**Client configuration**

Client version: 3.1
Operating System: Linux Mint 20, Mate edition, 64-bit
OS language: german / deutsch
Kernel version: 5.4.0-58-generic
Qt version: 5.12.8
Client package: PPA deb http://ppa.launchpad.net/nextcloud-devs/client/ubuntu focal main

**Server configuration**

Nextcloud Version: 20.0.3.2, installed with NextCloudPi on a Raspberry Pi 3A+

**Log**

Log file available, i don't know where to upload the 41 kb file

**Comment by sercxanto** [comment]
Same here with Ubuntu 20.04 with all updates applied and nextcloud PPA version 3.1.0-20201214.180315.7f4ffd0f7-1.0~focal1.

I am not able to start nextcloud-desktop, the client immedeately crashes at startup.

This is the backtrace from a gdb session, see [gdb_backtrace.txt](https://github.com/nextcloud/desktop/files/5721542/gdb_backtrace.txt):

```
Thread 1 "nextcloud" received signal SIGSEGV, Segmentation fault.
0x000055555571f1be in OCC::User::avatar() const ()
(gdb) bt
#0  0x000055555571f1be in OCC::User::avatar() const ()
#1  0x000055555571f2b9 in OCC::UserModel::avatarById(int const&) ()
```

The last log lines of the client log also piint towards avatar handling:

```
2020-12-20 22:01:18:503 [ debug nextcloud.sync.networkjob ]     [ OCC::AbstractNetworkJob::slotFinished ]:      Network job OCC::JsonApiJob finished for "ocs/v1.php/cloud/user"
2020-12-20 22:01:18:609 [ info nextcloud.sync.credentials.webflow ]:    request finished
2020-12-20 22:01:18:612 [ debug nextcloud.sync.networkjob.avatar ]      [ OCC::AvatarJob::finished ]:   Retrieved Avatar pixmap!
```

**Comment by er-vin** [comment]
Duplicate of #2710 fix is produced and will be released with 3.1.1. In the meantime you can use the AppImage on #2713.
